plugins/node/opentelemetry-instrumentation-net/test/connect.test.ts

```diff
@@ -0,0 +1,257 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { SpanKind, SpanStatusCode } from '@opentelemetry/api';
+import {
+  InMemorySpanExporter,
+  ReadableSpan,
+  SimpleSpanProcessor,
+} from '@opentelemetry/tracing';
+import { GeneralAttribute } from '@opentelemetry/semantic-conventions';
+import * as assert from 'assert';
+import { NodeTracerProvider } from '@opentelemetry/node';
+import { NetInstrumentation } from '../src/net';
+import * as net from 'net';
+import * as os from 'os';
+import * as path from 'path';
+
+const memoryExporter = new InMemorySpanExporter();
+const provider = new NodeTracerProvider();
+provider.addSpanProcessor(new SimpleSpanProcessor(memoryExporter));
+
+function assertClientSpan(span: ReadableSpan) {
+  assert.strictEqual(span.kind, SpanKind.CLIENT);
+}
+
+function assertAttrib(span: ReadableSpan, attrib: string, value: any) {
+  assert.strictEqual(span.attributes[attrib], value);
+}
+
+describe('NetInstrumentation', () => {
+  const PORT = 42123;
```

Comment:
can you please move const and functions outside of describe block - const to top, functions below or in separate `utils.ts` file (one or two are fine, otherwise would prefer moving them to separate file)